worker/sync/db-persist/spoor-publish.js

```diff
@@ -2,7 +2,7 @@
 
 const path = require('path');
 
-const { default: log } = require('@financial-times/n-logger');
+const log = require('../../../server/lib/logger');
```

Comment:
wouldnt it be better to change the affected place, rather than updating all loggers?
do we apply same practice on all our services?

Reply:
Not sure I understand what you mean, sorry...

Reply:
you are changing this bcause of leaked pwd, right?
if so, why not simply change that place and leave rest as it was

Reply:
Because it's not just passwords that are leaking and using `n-mask-logger` will handle all the other cases as well (including places where we can't tell via the code that PII is leaking).